### PR TITLE
Feature/iotcore 251 c

### LINKED ISSUE #41: create a version reponse - Perseo-fe
Create a version request (http) or message in log to Perseo-fe.
examples: 
```
http://host:port/version
```
```
In log when start Perseo
```

### PR DESCRIPTION
**Pull request by crbrox**
Add /version path to server for retrieving the version of the component

@fgalan @dmoranj  @iariasleon 

Closes #41 
IOTCORE-251

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete branch)

File: lib/routes/versionController.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * please contact with::[contacto@tid.es]
+ */
+'use strict';
+
+var config = require('../../config'),
+    fs = require('fs'),
+    myutils = require('../myutils'),
+    packageText, packageObj, versionObj;
+
+packageText = fs.readFileSync('package.json');
```
Comment (dmoranj) [inline lib/routes/versionController.js]: Why don't you `require('package.json')`? That way you wouldn't need the two steps, and you can avoid having executable code out of a function in the code (that seems a bit weird.

Comment (crbrox) [inline lib/routes/versionController.js]: Fixed in 575770b

File: lib/routes/versionController.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * please contact with::[contacto@tid.es]
+ */
+'use strict';
+
+var config = require('../../config'),
+    myutils = require('../myutils'),
+    packageObj = require('../../package.json'),
+    versionObj;
+
+versionObj = {
```
Comment (dmoranj) [inline lib/routes/versionController.js]: You could make this assignation in the variable declaration two lines avobe.

Comment (crbrox) [inline lib/routes/versionController.js]: Fixed in 0bde7de

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**